src/WebJobs.Script/Workers/WorkerConsoleLogService.cs

```diff
@@ -62,16 +59,7 @@ internal async Task ProcessLogs()
                 while (await source.OutputAvailableAsync(_cts.Token))
                 {
                     var consoleLog = await source.ReceiveAsync();
-
-                    if (WorkerProcessUtilities.IsToolingConsoleJsonLogEntry(consoleLog))
```

Comment:
I think we added this case in #8435 Would that use case break now?

Reply:
https://github.com/Azure/azure-functions-dotnet-worker/issues/900

Reply:
should be the same behavior above (same logger category, too)... we're just moving it up a level so that it happens at the WebHost rather than JobHost

Reply:
(but @soninaren -- you should double-check that it still works)

Reply:
I tested it end to end, it works as expected